pandas/core/arrays/integer.py

```diff
@@ -696,22 +696,77 @@ def integer_arithmetic_method(self, other):
 """
 
 # create the Dtype
-_dtypes = {}
-for dtype in ['int8', 'int16', 'int32', 'int64',
-              'uint8', 'uint16', 'uint32', 'uint64']:
-
-    if dtype.startswith('u'):
-        name = "U{}".format(dtype[1:].capitalize())
-    else:
-        name = dtype.capitalize()
-    classname = "{}Dtype".format(name)
-    numpy_dtype = getattr(np, dtype)
-    attributes_dict = {'type': numpy_dtype,
-                       'name': name,
-                       '__doc__': _dtype_docstring.format(dtype=dtype)}
-    dtype_type = register_extension_dtype(
-        type(classname, (_IntegerDtype, ), attributes_dict)
-    )
-    setattr(module, classname, dtype_type)
-
-    _dtypes[dtype] = dtype_type()
+Int8Dtype = register_extension_dtype(
+    type('Int8Dtype', (_IntegerDtype, ), {
+        'type': getattr(np, 'int8'),
```

Comment:
No need to use getattr for these just access normally

Reply:
just `np.int8` ?